lib/install/config/webpacker.yml

```diff
@@ -33,7 +33,7 @@ development:
     host: localhost
     port: 8080
     https: false
-    hot_reloading: false
+    hmr: false
```

Comment:
I think `hmr` is consistent with names like 'WEBPACKER_HMR'

Also, considering usages like ` Webpacker::DevServer.hmr?` and def `hmr?`, they are consistent.